<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>events</class>
 <widget class="QWidget" name="events">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eve">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);

font: 14pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>打开环境检测</string>
   </property>
  </widget>
  <widget class="QLabel" name="cam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(128, 159, 225);
border-radius: 20px;
background: rgba(0,0,0,50);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fan">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>打开风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alarm">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>打开警报</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);


background: rgba(0,0,0,0);
border-radius: 16px;
</string>
   </property>
   <property name="text">
    <string>访客检测窗口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeFDiagCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
color: rgba(255, 255, 255);
font: 14pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,50);
border-radius: 16px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
color: rgba(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgb(0,0,0,50);
border-radius: 16px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,0);
</string>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>湿度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
border-radius: 16px;
font: 20pt &quot;Ubuntu&quot;;
background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>FIRE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>851</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/back8.png);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/feng.png);
image: url(:/feng.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/temp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/fire.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>connect</zorder>
  <zorder>lineEdit</zorder>
  <zorder>eve</zorder>
  <zorder>cam</zorder>
  <zorder>fan</zorder>
  <zorder>alarm</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
